server/auth.go

```diff
@@ -26,9 +26,7 @@ func (s Server) Login(writer http.ResponseWriter, request *http.Request) {
 		u.SendError(writer, status)
 		return
 	}
-
-	resp := auth.Authenticate(account.Login, account.Password, s.LDAP)
-	u.SendResponse(writer, resp)
+	auth.Authenticate(account.Login, account.Password, s.LDAP, writer)
```

Comment:
I think it's a not very good idea to move sending response to auth function, because if we keep sending only in one place, like controllers code will be more clear and readable.. Maybe will be better return error too, like it done in most of Go functions.

Reply:
Again it seemed to me more straightforward, but OK.